notebooks/read_cube.ipynb

```diff
@@ -406,8026 +805,35 @@
-       "   -0.034240800379809366,\n",
-       "   -0.03433420143447536,\n",
-       "   -0.034426899589041625,\n",
-       "   -0.034519899812559734,\n",
-       "   -0.034613099955154304,\n",
-       "   -0.03470540027476643,\n",
-       "   -0.03479840159738594,\n",
-       "   -0.03489100069389597,\n",
-       "   -0.03498380082411378,\n",
-       "   -0.035077100979100745,\n",
-       "   -0.035169699906488334,\n",
-       "   -0.03526290041488393,\n",
-       "   -0.03535519930655597,\n",
-       "   -0.03544840163638639,\n",
-       "   -0.03554140111687647,\n",
-       "   -0.03563410113672742,\n",
-       "   -0.03572709973223738,\n",
-       "   -0.03582000083922514,\n",
-       "   -0.035912300955443434,\n",
-       "   -0.036005500251474816,\n",
-       "   -0.036098999312944795,\n",
-       "   -0.036191401173105014,\n",
-       "   -0.036284299970488015,\n",
-       "   -0.0363771016076765,\n",
-       "   -0.03646909969545234,\n",
-       "   -0.036563001142723595,\n",
-       "   -0.03665579964397945,\n",
-       "   -0.0367488001899585,\n",
-       "   -0.03684159967377711,\n",
-       "   -0.036934500212856734,\n",
-       "   -0.0370274002435698,\n",
-       "   -0.0371202003142794,\n",
-       "   -0.03721279939119501,\n",
-       "   -0.0373055995485279,\n",
-       "   -0.0373978999192689,\n",
-       "   -0.03749090072842102,\n",
-       "   -0.03758440017663292,\n",
-       "   -0.037677200042575,\n",
-       "   -0.037770101345022865,\n",
-       "   -0.03786279990708008,\n",
-       "   -0.037955100108815254,\n",
-       "   -0.03804840051397526,\n",
-       "   -0.03814140109742733,\n",
-       "   -0.03823480134837338,\n",
-       "   -0.03832700073530056,\n",
-       "   -0.0384200003823923,\n",
-       "   -0.03851269943011353,\n",
-       "   -0.0386059002141548,\n",
-       "   -0.03869820061769874,\n",
-       "   -0.03879130024023394,\n",
-       "   -0.0388845008432102,\n",
-       "   -0.038977400247294805,\n",
-       "   -0.03907089956870592,\n",
-       "   -0.03916360122687886,\n",
-       "   -0.039257301089914945,\n",
-       "   -0.0393500006149653,\n",
-       "   -0.03944339953101695,\n",
-       "   -0.03953630097036634,\n",
-       "   -0.0396300004075632,\n",
-       "   -0.03972290107625702,\n",
-       "   -0.03981470166993588,\n",
-       "   -0.03990829916889257,\n",
-       "   -0.04000080038889798,\n",
-       "   -0.04009390136682353,\n",
-       "   -0.04018630136924827,\n",
-       "   -0.04027920031589061,\n",
-       "   -0.04037259980672138,\n",
-       "   -0.04046460035258563,\n",
-       "   -0.040557999347227695,\n",
-       "   -0.0406503006491793,\n",
-       "   -0.04074340182162566,\n",
-       "   -0.04083590103936866,\n",
-       "   -0.040928700113815566,\n",
-       "   -0.041021601072140215,\n",
-       "   -0.04111440054568199,\n",
-       "   -0.041208100159118376,\n",
-       "   -0.04129979966814669,\n",
-       "   -0.04139300041015805,\n",
-       "   -0.04148590064813
```

Comment:
Is there a way to clear these outputs before committing? ...Just takes up so much space...

Reply:
Yeah, in the future we need to keep limited output in the cubes, in this case the notebook only existed for the sake of testing and is gone now.